src/ble/BLEEndPoint.cpp

```diff
@@ -539,12 +539,6 @@ BLE_ERROR BLEEndPoint::Init(BleLayer * bleLayer, BLE_CONNECTION_OBJECT connObj,
     VerifyOrExit(connObj != BLE_CONNECTION_UNINITIALIZED, err = BLE_ERROR_BAD_ARGS);
     VerifyOrExit((role == kBleRole_Central || role == kBleRole_Peripheral), err = BLE_ERROR_BAD_ARGS);
 
-    // Null-initialize callbacks and data members.
-    //
-    // Beware this line should we ever use virtuals in this class or its
-    // super(s). See similar lines in chip::System::Layer end points.
-    memset(this, 0, sizeof(*this));
```

Comment:
All members are in fact explicitly initialized below. (The change to `PacketBufferHandle` makes the compiler reject this, although it was already non-conforming anway.)